### PR TITLE
Reorganise extension part 2: publish section

### PR DESCRIPTION
**Pull request by gtardif**
### Proposed changes

Reorganise extension publish section with the new share functionality, improved label description

### CODE REVIEW

File: desktop/extensions-sdk/extensions/DISTRIBUTION.md [hunk 1]
```diff
@@ -1,54 +1,52 @@
...
+Any new image pushed to an image repository corresponding to a Docker Extension defines a new version of that extension. Image tags are used to identify version numbers. Extension versions must follow semver to make it easy to understand and compare versions.
 
-Users can download and install the newer version of an extension without updating Docker Desktop itself.
+Docker Desktop scans the list of extensions published in the marketplace for new versions, and provides notifications to users when they can upgrade a specific extension. Extensions that aren't part of the marketplace don't have automatic update notifications at the moment.
 
-## API dependencies
+Users can download and install the newer version of any extension without updating Docker Desktop itself.
 
-Extensions must specify the Extension API version they rely on. Currently there is no technical validation of this version, as the extension framework is still experimental.
+## API dependencies
```
Comment (benja-M-1) [inline desktop/extensions-sdk/extensions/DISTRIBUTION.md:43]: **suggestion**: be more explicit about what API we are talking about. I used to think it was about API in general, so I was confused.
```suggestion
## Extensions API dependencies
```

File: desktop/extensions-sdk/extensions/index.md [hunk 2]
```diff
@@ -1,76 +1,43 @@
...
-    > For Docker Extensions images to be listed in Docker Desktop, they must be approved by Docker and the tags must follow semantic versioning, e.g: `0.0.1`.
-    > 
-    > See [distribution and new releases](DISTRIBUTION.md#distribution-and-new-releases) for more information.
-    > 
-    > See [semver.org](https://semver.org/){:target="_blank" rel="noopener" class="_"} to learn more about semantic versioning.
+## Promote your extension
 
-2. Push the image to Docker Hub:
+Once your extension is available on Docker Hub, users who have access to the extension image can install it using the Docker CLI.
+However, you might want better way make your extension visible and easy to install than copy pasting CLI commands in a terminal.
```
Comment (benja-M-1) [inline desktop/extensions-sdk/extensions/index.md:24]:
```suggestion
However, you might want a better way to make your extension visible and easy to install than copy-pasting CLI commands in a terminal.
```

File: desktop/extensions-sdk/extensions/publish.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+If you want to publish your extension in the Marketplace, you can submit your extension [here](https://www.docker.com/products/extensions/submissions/){: target="_blank" rel="noopener" class="_" }.
+
+All extensions submitted to the Extension Marketplace are reviewed and approved by the Docker Extension team before listing. This review process ensures a level of trust, security, and quality for developers using Docker Extensions and allows for extension developers to get feedback.
+
+### Before you submit
+
+Ensure your extension has followed the guidelines outlined in this section before submitting for your extension for review. Docker highly encourage you to check the guidelines as not doing so may considerably impact the duration of the approval process.
+
+These guidelines don't replace Docker terms of service or guarantee approval. As the Extension Marketplace continues adding new features for both Extension users and publishers, expect that your maintain your extension over time to ensure it stays available in the Marketplace.
```
Comment (benja-M-1) [inline desktop/extensions-sdk/extensions/publish.md:19]:
```suggestion
These guidelines don't replace Docker's terms of service or guarantee approval. As the Extension Marketplace continues adding new features for both Extension users and publishers, expect that you maintain your extension over time to ensure it stays available in the Marketplace.
```

**Review by benja-M-1 — CHANGES_REQUESTED** [review]
I selected the wrong review state on my previous review...

File: desktop/extensions-sdk/extensions/DISTRIBUTION.md [hunk 4]
```diff
@@ -1,54 +1,52 @@
...
+Extension images must be multi-arch images so that users can install extensions on ARM/AMD hardware. These multi-arch images can include ARM/AMD specific binaries. Mac users will automatically use the right image based on their architecture.
+Extensions that install binaries on the host must also provide Windows binaries in the same extension image.
 
-Docker Desktop includes an allow-list of extensions available to users. The extension image Hub repository (like `mycompany/my-desktop-extension`) must be part of the Docker Desktop allow-list to be recognized as an extension.
+> See how to [build a multi-arch image](multi-arch.md) for your extension.
 
-This allow-list specifies which Hub repositories are to be used by Docker Desktop to download and install extensions with a specific version at a given point in time.
+You can implement extensions without any constraints on the code repository. Docker doesn't need access to the code repository in order to use the extension. You can entirely manage new releases of your extension, without any dependency on Docker or Docker Desktop.
```
[the remark this answers is not in the document]
Comment (gtardif) [reply desktop/extensions-sdk/extensions/DISTRIBUTION.md:31]: Yes, but I'll make this more explicit

**Review by benja-M-1 — APPROVED** [review]
Great 👍

**Pull request merged**